### ISSUE TITLE
Kubernetes service multiple apiserver endpoints

### ISSUE THREAD

**Issue by ramonrue** [kubernetes/kubernetes#19989]
I'm having problems with multiple kubernetes apiserver endpoints. I set them up like described here #19161 . After that, it more or less works. One problem remains:
When executing a ```kubectl get ep``` I get the following:
```shell
NAME         ENDPOINTS          AGE
kubernetes   192.168.1.32:443   2d
```
Executing the same command a few seconds later, gives me:
```shell
NAME         ENDPOINTS          AGE
kubernetes   192.168.1.31:443   2d
```

The three apiservers (Ip Adresses .31,.32,.33) are constantly overwriting the etcd-key ```/registry/services/endpoints/default/kubernetes```
This confuses the haproxy for example.
Would it be possible to write multiple endpoints to the kubernetes-service?
Or can I somehow set a domain name as an endpoint? If yes, which file do I have to edit?

Working on CoreOS Alpha 933.0.0

Ok, seen in #18628

**Comment by thockin** [comment]
Did you set the `--apiserver-count` flag to 3?

**Comment by ramonrue** [comment]
I'm using Kubernetes v1.1.2, is this feature already implemented?

**Comment by thockin** [comment]
Great question!  I thought that had made it in for 1.1 but I see that it
did not.  In that case, the answer is that you can't.  1.1 Assumes a single
master endpoint.

**Comment by ramonrue** [comment]
Can I somehow use an alpha-version of the hyperkube-container?
Tried with gcr.io/google_containers/hyperkube:v1.2.0-alpha.1 to alpha.6, but I always got "Tag not found".
Also I'm wondering;
#18628 says that the commit got merged into kubernetes:master, but I can't find it. I also can't find it in all alpha-versions, just in Tree a09d85b the feature seems to be implemented.
And that's why I'm wondering even more because nothing can be found in the alpha-versions...

**Comment by ramonrue** [comment]
Okay I think I did it.
First I downloaded the source code of the Tree a09d85b on a CentOS System.
With ```make``` I created the packages and then copied the ```"kube-apiserver"``` under ```./_output/local/bin/linux/amd64/``` to the CoreOS-Master-Servers (the apiservers). 
I started the kubelet normally (with the v1.1.2 Containers) and copied the kube-apiserver executable to the root-directory of the apiserver-container. 
Then I commited the container;
```shell
docker commit [id] gcr.io/google_containers/hyperkube:v.1.2-alpha
```
After the commit I changed the "command" section in the file ```/etc/kubernetes/manifests/kube-apiserver``` to this:
```shell
[...]
   command:
    - /kube-apiserver
    - --apiserver-count=3
[...]
```
I saved the file, stopped the kubelet.service, killed all running docker-containers and started the kubelet again. Now the kubelet started the new docker container with the correct executable.
And see there:
```shell
kubectl get ep
NAME         ENDPOINTS                                            AGE
kubernetes    [IP-1]:443,[IP-2]:443,[IP-3]:443                    3d
```

This will just be a temporary solution until v1.2 will be on CoreOS.
